mypy 1.15.0 fails in CI but not locally

Fails with:
```
Run mypy --version && mypy
mypy 1.15.0 (compiled: yes)
src/bokeh/server/tornado.py:459: error: Unused "type: ignore" comment 
[unused-ignore]
            super().__init__(all_patterns, # type: ignore[arg-type] # TODO...
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~...
Found 1 error in 1 file (checked 660 source files)
```

--- a/src/bokeh/server/tornado.py
+++ b/src/bokeh/server/tornado.py
@@ -456,9 +456,7 @@ class BokehTornado(TornadoApplication):
         for line in pformat(all_patterns, width=60).split("\n"):
             log.debug("  " + line)
 
-        super().__init__(all_patterns, # type: ignore[arg-type] # TODO: this may be another bug in mypy (not sure; but looks suspicious)
-            websocket_max_message_size=websocket_max_message_size_bytes,
-            **kwargs)
+        super().__init__(all_patterns, websocket_max_message_size=websocket_max_message_size_bytes, **kwargs)
 
     def initialize(self, io_loop: IOLoop) -> None:
         ''' Start a Bokeh Server Tornado Application on a given Tornado IOLoop.

